app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/TransformAdd.tsx

```diff
@@ -39,6 +40,17 @@ export default function TransformAddButton({
   const open = !!anchorEl;
   const subMenuOpen = !!subMenuAnchorEl;
 
+  const getCurrentColumnName = (columnName: string) => {
```

Comment:
can we move this out of here and just pass the current column name to TransformAdd - that way you don't have to pass all of the transformations to that component